packages/wallet-connectors/src/WalletConnection.ts

```diff
@@ -69,7 +70,7 @@ export interface WalletConnection {
         accountAddress: string,
         type: AccountTransactionType.Update | AccountTransactionType.InitContract,
         payload: SendTransactionPayload,
-        parameters: Record<string, unknown>,
+        parameters: SmartContractParameters,
```

Comment:
Please add this interface change to the changelog. Is it a breaking change @shjortConcordium ?